pkgs/tools/security/signify/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchurl, libbsd, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  name = "signify-${version}";
+  version = "v24";
+
+  src = fetchurl {
+    url = "https://github.com/aperezdc/signify/archive/${version}.tar.gz";
+    sha256 = "0594vyvkq176xxzaz9xbq8qs0xdnr8s9gkd1prblwpdvnzmw0xvc";
+  };
+
+  doCheck = true;
+
+  buildInputs = [ libbsd pkgconfig ];
+
+  preInstall = ''
+    export PREFIX=$out
+  '';
+
+  meta = with stdenv.lib; {
+    description = "OpenBSD signing tool";
+    longDescription = ''
+      OpenBSDs signing tool.
```

Comment:
No need for the `longDescription` if it is the same text as the short `description`

Reply:
I decided to embellish the long description to add more detail, since that was my original intention.